### PR TITLE
Improves bubble alerts

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Adds punctuation, fixes grammar, makes em stay on screen longer.

## Why It's Good For The Game

Grammar and grammar good, being able to read the bubble alerts good.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
tweak: bubble alerts stay on screen longer
spellcheck: fixed punctuation and grammar in bubble alerts
/:cl:

### CODE REVIEW

File: code/game/objects/structures/girders.dm [hunk 1]
```diff
@@ -217,43 +217,43 @@
...
 
 // Wirecutter behavior for girders
 /obj/structure/girder/wirecutter_act(mob/user, obj/item/tool)
 	. = FALSE
 	if(state == GIRDER_REINF_STRUTS)
-		balloon_alert(user, "You start removing the inner grille")
+		balloon_alert(user, "You start removing the inner grille...")
 		if(tool.use_tool(src, user, 40, volume=100))
-			balloon_alert(user, "Inner grille removed")
+			balloon_alert(user, "You remove the inner grillie.")
```
Comment (Mat05usz) [inline code/game/objects/structures/girders.dm]: Grille not grillie

Comment (ivanmixo) [inline code/game/objects/structures/girders.dm]:
```suggestion
			balloon_alert(user, "You remove the inner grille.")
```

**Review by PowerfulBacon — COMMENTED** [review]
Can you also give them the correct appearance flags so resizing doesn't affect them?

File: code/game/objects/items/melee/energy.dm [hunk 2]
```diff
@@ -146,7 +146,7 @@
 		var/obj/item/stock_parts/cell/C = R.cell
 		if(active && !(C.use(hitcost)))
 			attack_self(R)
-			balloon_alert(R, "Out of charge")
+			balloon_alert(R, "Your energy sword is out of charge.")
```
Comment (PowerfulBacon) [inline code/game/objects/items/melee/energy.dm]:
```suggestion
			balloon_alert(R, "Your [name] is out of charge.")
```

**Review by PowerfulBacon — COMMENTED** [review]
Can you post a gif/video of what the new fade out looks like?

**Comment by ivanmixo** [comment]
https://user-images.githubusercontent.com/22431091/167688446-959eabac-fcd7-4f25-bb32-986c4ec531fe.mp4

**Comment by Fikou** [comment]
dont make a feature entirely pointless by cutting its purpose challenge (impossible)

**Comment by ivanmixo** [comment]
What did he mean by this

**Review by PowerfulBacon — APPROVED** [review]
Good change, hopefully we can get rid of these entirely eventually

**Pull request merged**